src/test/java/spoon/test/intercession/IntercessionTest.java

```diff
@@ -166,6 +163,8 @@ public boolean matches(CtMethod<?> element) {
 				});
 
 		for (CtMethod<?> setter : setters) {
+			if (setter.getSimpleName().toLowerCase().contains("metadata")) continue;
```

Comment:
On Wed, Feb 10, 2016 at 10:06 AM, Gérard Paligot <notifications@github.com>
wrote:

> > +			if (setter.getSimpleName().toLowerCase().contains("metadata")) continue;
>
>
I'll accept the PR, but I don't understand why the test fails if I remove
that guard.